.github/workflows/build.yml

```diff
@@ -32,6 +32,10 @@ jobs:
             spring-boot-version: 3.3.0
             spring-boot-display-version: 3.3.x
             experimental: false
```

Comment:
I this file shouldn't we remove the 3.2.x from the GitHub action since it is no longer supported.

Reply:
I think we should